htdocs/comm/propal/class/propaleligne.class.php

```diff
@@ -343,6 +343,10 @@ class PropaleLigne extends CommonObjectLine
 	 */
 	public $multicurrency_total_ttc;
 
+	/**
+	 * @var string Serialized subtotal options
+	 */
+	public mixed $subtotal_options;
```

Comment:
Attention, tu introduis des incompatibilités avec des versions de PHP toujours supportées par Dolibarr (PHP 7.2 minimum, où le typage des propriétés n'existe pas).

```suggestion
	public $subtotal_options = [];
```